src/igvfd/schemas/changelogs/file_set.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`file_set.json`*
 
+### Minor changes since schema version 2
```

Comment:
I thought we are converting changelogs to be date based? or is the assumption that the other ticket will cover this change? IGVF-417?

Reply:
I was not sure which ticket would proceed first, added this line assuming I would have to redo all the changes in rebasing.